contrib/ruby/ext/trilogy-ruby/cext.c

```diff
@@ -96,6 +96,7 @@ static void free_trilogy(struct trilogy_ctx *ctx)
     if (ctx->conn.socket != NULL) {
         trilogy_free(&ctx->conn);
     }
+    free(ctx);
```

Comment:
I think this should be Ruby's `xfree`

Reply:
👍🏻 Thanks! I can't say I fully understand why yet, but I see that it's doing a whole bunch of extra stuff.